lib/rubocop/ast/processed_source.rb

```diff
@@ -95,8 +95,16 @@ def blank?
         ast.nil?
       end
 
+      # @return [Boolean] if the given line number has a comment.
+      def commented_line?(line)
```

Comment:
I think the name is a bit misleading (it sounds like a line that was commented out), but it's a good change. I guess you just tried to mimic the `commented?` name we've employed before. Perhaps it should have been something like `contains_comments?` or whatever.

Reply:
I agree, I should have thought of a better name, especially that `util` in `rubocop` has `comment_line?` which is true only for full lines.
So which is best: `contains_comment?`, or `line_with_comment?`.

Reply:
You can make a method `contains_comment?` that acts differently on line numbers and source ranges. :wink: If not - I'd go with `line_with_comment?`, as I suggested `contains_comment?` as an alternative name for `commented?`.

Reply:
Updated with `line_with_comment?` and `contains_comment?` (with deprecated alias `commented?`)